worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -140,6 +142,7 @@ private MemoryManager(CelebornConf conf) {
             if (lastState != servingState) {
               logger.info("Serving state transformed from {} to {}", lastState, servingState);
               if (servingState == ServingState.PUSH_PAUSED) {
+                lastPauseTime = System.currentTimeMillis();
```

Comment:
PUSH_AND_REPLICATE_PAUSED will pause push data and Celeborn worker might changed to PUSH_AND_REPLICATE_PAUSED directly is memory pressure is high.

Reply:
Got this. After the logic here clearly #1835 , will go on this pr.